accelerator/config.c

```diff
@@ -103,6 +126,19 @@ status_t cli_core_set(ta_core_t* const core, int key, char* const value) {
       }
       iota_service->http.port = ta_conf->iota_port_list[0];
       break;
+
+    case CA_PEM:
+      if ((file = fopen(value, "r")) == NULL) {
+        /* The specified configuration file does not exist */
+        ta_log_error("%s\n", ta_error_to_string(SC_CONF_FOPEN_ERROR));
+        return SC_CONF_FOPEN_ERROR;
+      }
+      char* temp_str = (char*)malloc(sizeof(char) * (CA_PEM_LEN + 1));
+      fread(temp_str, CA_PEM_LEN, 1, file);
+      iota_service->http.ca_pem = temp_str;
```

Comment:
`ca_pem` should be deallocated after no further use.

Reply:
However, we need to assign `ca_pem` every time we allocate a service object.